linux_os/guide/system/accounts/accounts-physical/require_singleuser_auth/bash/shared.sh

```diff
@@ -1,10 +1,22 @@
-# platform = multi_platform_rhel
+# platform = multi_platform_rhel, multi_platform_fedora
 
 {{% if init_system == "systemd" -%}}
+
+{{% if product == "fedora" -%}}
+service_file="/usr/lib/systemd/system/rescue.service"
+sulogin="/usr/lib/systemd/systemd-sulogin-shell"
+if grep "^ExecStart=.*" "$service_file" ; then
+    sed -i "s%^ExecStart=.*%ExecStart=-$sulogin rescue%" "$service_file"
+else
+    echo "ExecStart=-$sulogin rescue" >> "$service_file"
+fi
+{{%- else -%}}
 grep -q "^ExecStart=\-.*/sbin/sulogin" /usr/lib/systemd/system/rescue.service
```

Comment:
can you reuse `service_file` variable? (Possibly change that one and `sulogin` to CONSTANTS) The logic difference is now looks bigger than it really is.